matchbook-sdk-rest/src/main/java/com/matchbook/sdk/rest/AbstractRestClient.java

```diff
@@ -5,29 +5,29 @@
 import java.util.stream.Collectors;
 
 import com.matchbook.sdk.core.StreamObserver;
-import com.matchbook.sdk.rest.dtos.RestRequest;
-import com.matchbook.sdk.rest.dtos.RestResponse;
+import com.matchbook.sdk.core.exceptions.MatchbookSDKHttpException;
 import com.matchbook.sdk.rest.configs.HttpCallback;
 import com.matchbook.sdk.rest.configs.Serializer;
-import com.matchbook.sdk.core.exceptions.MatchbookSDKHttpException;
+import com.matchbook.sdk.rest.dtos.RestRequest;
+import com.matchbook.sdk.rest.dtos.RestResponse;
 
 abstract class AbstractRestClient {
 
-    private final ConnectionManager connectionManager;
+    protected final ConnectionManager connectionManager;
 
     protected AbstractRestClient(ConnectionManager connectionManager) {
         this.connectionManager = connectionManager;
     }
 
     protected <REQ extends RestRequest, RESP extends RestResponse<T>, T>
-            void getRequest(String url, REQ request, StreamObserver<T> observer, Class<RESP> responseClass) {
+    void getRequest(String url, REQ request, StreamObserver<T> observer, Class<RESP> responseClass) {
```

Comment:
I think removing the indentation is not a good idea because it would make the code less readable.